### PR TITLE
restructure_docs

### PR DESCRIPTION
**Pull request by laisbsc**
PR for docs restructure. Ready for review in parts.

For now, the aim of this PR is to:
* [x] Reformat README.md at [GH](https://github.com/Quansight/qhub/)
* [ ] Restructure the documentation;
* [ ] Change theme;
* [ ] Bulk-up FAQ.
    * add #167 to it.
    * add #242 to it.

### CODE REVIEW

**Review by kcpevey — COMMENTED** [review]
I wrote these this morning, but just realize I must have never clicked "submit"! I hope they aren't out of date now!

File: README.md [hunk 1]
```diff
@@ -1,104 +1,195 @@
...
-Numba. For more information see https://www.quansight.com/about-us.
+QHub is an open source tool that enables users to build and maintain
+cost-effective and scalable compute/data science platforms [on-premises](#on-premises) or on [cloud providers](#cloud-providers).
+with minimal DevOps experience.
 
-## Usage
-
-`qhub` is installed as a command line application in Python. It requires you to choose your the cloud provider you desire. once you've decided on a provider `qhub` will walk you through the following steps to configure your deployment:
+## On-Premises
+QHub On-Prem is based on OpenHPC.
```
[the remark this answers is not in the document]
Comment (laisbsc) [reply README.md]: @dharhas do you have any thoughts on this? Shall we rename it?

File: docs/docs/do/configuration.md [hunk 2]
```diff
@@ -157,12 +164,14 @@ google_cloud_platform:
...
-### Amazon Web Services
+#### Amazon Web Services (AWS)
 
-Amazon Web Services has similar support to Digital Ocean and does not
-allow auto-scaling below 1 node. Additionally AWS requires that a
-kubernetes cluster run in two availability zones. [Consult aws instance
-types for possible options.](https://aws.amazon.com/ec2/instance-types/)
+Amazon Web Services has similar support to DigitalOcean and does not
+allow auto-scaling below 1 node. Additionally, AWS requires that a
+Kubernetes cluster run in two Availability Zones (AZs).
```
[the remark this answers is not in the document]
Comment (laisbsc) [reply docs/docs/do/configuration.md:171]: Added. And this is now a question at the FAQ as well.
TY! 🥇

File: docs/docs/do/configuration.md [hunk 3]
```diff
@@ -459,13 +470,14 @@ environments:
 
 2. Put configuration file in directory in name `qhub-config.yaml`
 
-3. Initialize the qhub repository
+~2. Put configuration file in the directory with name `qhub-config.yaml` [???]
```
[the remark this answers is not in the document]
Comment (laisbsc) [reply docs/docs/do/configuration.md:473]: An unfinished sentence on the original docs. I left the question marks to signal the need to fix. I'm currently working on pushing the README, then I will get to this one.

You have a great eye for revision, though! Thanks 👏🏽

File: docs/docs/faqs.md [hunk 4]
```diff
@@ -1,24 +1,135 @@
...
+> 
+> Have feedback or would like to request improvements on the docs? Open an [issue](https://github.com/Quansight/qhub/issues/new/choose) and tell us more.
+    
+
+### Who was QHub made for?
+In summary, anyone who would like to deploy using JupyterHub. From data scientists and researchers to DevOps engineers,
+QHub was designed to abstract the complexities involved on using cloud services and parallel computing.
+
+It introduces the concept of deployment from JupyterHub to a chosen Cloud provider using Kubernetes, without the need for 
+a DevOps specialist. I.e., _Kubernetes for those who don't know how to use Kubernetes_.
```
[the remark this answers is not in the document]
Comment (laisbsc) [reply docs/docs/faqs.md:15]: I'm grateful for the nitpickers in my life, especially when they revise my work.

Thank you!

File: docs/docs/do/configuration.md [hunk 5]
```diff
@@ -305,18 +315,19 @@ environments:
...
 each environment include `ipykernel`, `ipywidgets`, `dask==2.14.0`,
 `distributed==2.14.0`, `dask-gateway==0.6.1`. Upon changing the
 environment definition expect 1-10 minutes upon deployment of the
 configuration for the environment to appear.
 
-# Example Full Configuration
+# Full Configuration Example
 
-Everything in the configuration is set besides
+Everything in the configuration is set besides [???]
```
[the remark this answers is not in the document]
Comment (laisbsc) [reply docs/docs/do/configuration.md:330]: see [this reply](https://github.com/Quansight/qhub/pull/228#discussion_r560910117).

**Pull request merged**